pkg/projects/Microsoft.NETCore.UniversalWindowsPlatform/project.json.template

```diff
@@ -0,0 +1,16 @@
+{
+  "dependencies": {
+    "Microsoft.Net.Native.Compiler": "1.6.0",
+    "Microsoft.Private.CoreFx.UAP": "4.4.0-preview1-25205-01",
```

Comment:
NETCore.App doesn't include WCF, its only part of UWP.  You should be including WCF here as a redist in the flat meta-package.